src/components/MoviesCardsContainer/MoviesCardsContainer.js

```diff
@@ -22,8 +22,8 @@ const MoviesCardsContainer = ({ allMovieData, displayDetails, sortByTitle, sortB
             <div className="sort">
                 <p className="sort-by">Sort by:</p>
                 <div className="sortButtons">
-                    <button className={sortByTitlePressed && 'pressed'} onClick={() => sortByTitle(allMovieData)} aria-pressed={sortByTitlePressed}>Title (A-Z)</button>
-                    <button className={sortByRatingPressed && 'pressed'} onClick={() => sortByRating(allMovieData)} aria-pressed={sortByRatingPressed}>Rating (Descending)</button>
+                    <button className={sortByTitlePressed ? 'pressed' : undefined} onClick={() => sortByTitle(allMovieData)} aria-pressed={sortByTitlePressed}>Title (A-Z)</button>
+                    <button className={sortByRatingPressed ? 'pressed' : undefined} onClick={() => sortByRating(allMovieData)} aria-pressed={sortByRatingPressed}>Rating (Descending)</button>
                 </div>
```

Comment:
I notice our Cypress testing has an issue now because we have a "cy.get" based on the '.sort-button-rating' class, which I actually got rid of in my accessibility branch. I think we can maybe just change the undefined on lines 25 and 26 to .sort-button-title and .sort-button-rating and it should fix the issue.

Reply:
I just pulled your branch down to test this and it does work:

`className={sortByTitlePressed ? 'pressed' : 'sort-button-title'}`
`className={sortByRatingPressed ? 'pressed' : 'sort-button-rating'}`

Reply:
Great catch! I can make this edit, test it on my end, and push back up.

Reply:
Just pushed back up with those changes -- the tests pass and everything is still working well on my end